### PR TITLE
create and list account tokens

### PR DESCRIPTION
**Pull request by bendecoste**

### CODE REVIEW

File: cmd/cape/cmd/token.go [hunk 1]
```diff
@@ -41,9 +42,176 @@ var tokenCmd = &cobra.Command{
...
+			return err
+		}
+
+		if resp.StatusCode != http.StatusOK {
+			cmd.SilenceUsage = true
+			if verbose {
+				fmt.Println("received status code", resp.StatusCode)
+			}
+
+			return fmt.Errorf("something went wrong")
```
Comment (justin1121) [inline cmd/cape/cmd/token.go:103]: We should have bit more info than this no? Or are we not returning anything yet?

File: cmd/cape/cmd/token.go [hunk 2]
```diff
@@ -41,9 +42,176 @@ var tokenCmd = &cobra.Command{
...
+			return err
+		}
+
+		if resp.StatusCode != http.StatusCreated {
+			cmd.SilenceUsage = true
+			if verbose {
+				fmt.Println("received status code", resp.StatusCode)
+			}
+
+			return fmt.Errorf("something went wrong, your token was not created")
```
Comment (justin1121) [inline cmd/cape/cmd/token.go:191]: Same thought as above.

**Review by justin1121 — COMMENTED** [review]
LGTM. Just a couple thoughts and your tests are failing :sweat_smile:

File: cmd/cape/cmd/token.go [hunk 3]
```diff
@@ -41,9 +42,176 @@ var tokenCmd = &cobra.Command{
...
+			return err
+		}
+
+		description, err := cmd.Flags().GetString("description")
+		if err != nil {
+			return err
+		}
+
+		if name == "" {
+			return fmt.Errorf("tokens must be alphanumeric")
```
Comment (justin1121) [inline cmd/cape/cmd/token.go:153]: Testing this out actually and found this message confusing

```suggestion
			return fmt.Errorf("token names must be alphanumeric")
```

File: cmd/cape/cmd/token.go (hunk 1 above)
Comment (bendecoste) [reply cmd/cape/cmd/token.go:103]: was just concerned that the server doesn't sanitize errors yet so it would show pg errors etc

Comment (justin1121) [reply cmd/cape/cmd/token.go:103]: Gotcha! All good.

**Review by justin1121 — DISMISSED** [review]

**Review by justin1121 — APPROVED** [review]

**Pull request merged**